src/wp-includes/widgets/class-wp-widget-media-gallery.php

```diff
@@ -248,6 +248,7 @@ public function render_control_template_scripts() {
 	protected function has_content( $instance ) {
 		if ( ! empty( $instance['ids'] ) ) {
 			$attachments = wp_parse_id_list( $instance['ids'] );
+			_prime_post_caches( $attachments );
```

Comment:
```suggestion
			_prime_post_caches( $attachments, false, false );
```

I don't think term and meta caches need to be primed here. Let's make this change.

Reply:
@spacedmonkey updated with needful changes. please take a look. Thanks